src/routes/blogs/+page.svelte

```diff
@@ -29,14 +29,24 @@
 			link: 'blogs/pytorch-on-the-edge',
 			image: 'https://onnxruntime.ai/_app/immutable/assets/pytorch-on-the-edge-with-ort.cdaa9c84.png'
 		},
+		{
+			title: 'Accelerating over 130,000 Hugging Face models with ONNX Runtime',
+			date: 'October 4th, 2023',
+			blurb:
+				'Learn more on how ONNX Runtime helps users accelerate open source machine learning models from Hugging Face.',
+			link: 'https://cloudblogs.microsoft.com/opensource/2023/10/04/accelerating-over-130000-hugging-face-models-with-onnx-runtime/',
```

Comment:
There's also a teaser blog hosted by HF that links to this one. Do you think the blogs page should link there or to this one?

Reply:
I think the blogs page should definitely link to the MSFT Open Source Blog post - it's quite a bit longer w/ some additional important info + also has a link to the Hugging Face teaser at the end to demonstrate that it was written in collaboration w/ them.